packages/pigeon/platform_tests/test_plugin/windows/pigeon/core_tests.gen.h

```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 //
-// Autogenerated from Pigeon (v5.0.1), do not edit directly.
+// Autogenerated from Pigeon (v6.0.0), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
 
-#ifndef PIGEON_CORE_TESTS_PIGEONTEST_H_
-#define PIGEON_CORE_TESTS_PIGEONTEST_H_
+#ifndef PIGEON_CORE_TESTS_GEN_H_CORE_TESTS_PIGEONTEST_H_
+#define PIGEON_CORE_TESTS_GEN_H_CORE_TESTS_PIGEONTEST_H_
```

Comment:
appears to be the namespace and the header file name combined

Reply:
which is exactly what the _getGuardName function does.

Reply:
So, this is the result of combining the CppOptions for both files, usually only namespace or the header file name is added per file. Now they're getting merged in the _getGuardName function.

pigeon_ + core_test.gen.h + core_tests_pigeontest + H_

Reply:
Not sure if it really matters, let me if I should change it somehow.

Reply:
We're only calling _getGuardName in the header, right? In which case namespace would never have been used in it, which makes sense because the namespace isn't part of the guard in Google C++ style.

It looks like having namespace in that function at all is just a latent bug that I missed in the initial C++ review, and that we never noticed because it was harmless. We should just remove that parameter and the associated code.